### PR TITLE
chore(documentation) Getting just the pure ID from curl with JQ

### PR DESCRIPTION
**Pull request by shanechambers-ds**
If you want to extend the get call for maybe creating an application deploy marker, the get from the list, and use the post for application deploy.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by nbaenam** [comment]
Hello @shanechambers-ds! Thank you so much for your feedback. Could you please sign the CLA? It's a required check and we can't merge this PR without signing. Thanks!

File: src/content/docs/apis/rest-api-v2/application-examples-v2/list-apps-host-ids-instance-ids.mdx [hunk 1]
```diff
@@ -25,6 +25,21 @@ curl -X GET 'https://api.newrelic.com/v2/applications.json' \
 -d "filter[name]=<var>${APP_NAME}</var>"
 ```
 
+## Get a Single application ID from a List filtered by application name [#get_app_id_from_name]
+
+You need to get an ID for an app that you have by name, or potentially do not have at all with a little you can pull just the id you need .
```
Comment (nbaenam) [inline src/content/docs/apis/rest-api-v2/application-examples-v2/list-apps-host-ids-instance-ids.mdx:30]: Could you please review this sentence? I don't understand it well. Is the word 'little' what you wanted to write?

Comment (shanechambers-ds) [reply src/content/docs/apis/rest-api-v2/application-examples-v2/list-apps-host-ids-instance-ids.mdx:30]: The part that should be there is 'jq'. I am fixing it right now. I've been offline most of the holiday.

Comment (nbaenam) [reply src/content/docs/apis/rest-api-v2/application-examples-v2/list-apps-host-ids-instance-ids.mdx:30]: OK, I'll merge it after this change. Thanks!

**Comment by nbaenam** [comment]
Hi @shanechambers-ds. Happy New Year!!! We have this pending PR. Could you please add the needed change? Thanks!

File: src/content/docs/apis/rest-api-v2/application-examples-v2/list-apps-host-ids-instance-ids.mdx (hunk 1 above)
Comment (nbaenam) [reply src/content/docs/apis/rest-api-v2/application-examples-v2/list-apps-host-ids-instance-ids.mdx:30]: Hi @shanechambers-ds! Could you please add the change to merge it? Thanks!

**Comment by nbaenam** [comment]
Hi @shanechambers-ds. Hope you're doing well. We have this pending PR from a long time ago. I remember that you need to change a sentence, right? Could you please add it or should I close this PR without merging it? Thanks!

**Comment by shanechambers-ds** [comment]
Sorry, I've been super busy. I will adjust it tonight.

*Shane Chambers | *DevOps

*(260) 602-7997 | *Denver, CO

***@***.*** ***@***.***>

**Comment by nbaenam** [comment]
Hi @shanechambers-ds, any update on this?

**Comment by nbaenam** [comment]
Hi @shanechambers-ds, any update on this?

File: src/content/docs/apis/rest-api-v2/application-examples-v2/list-apps-host-ids-instance-ids.mdx (hunk 1 above)
Comment (paperclypse) [inline src/content/docs/apis/rest-api-v2/application-examples-v2/list-apps-host-ids-instance-ids.mdx:30]:
```suggestion
Use this API call to get a single application ID from a list filtered by application name.
```

**Comment by nbaenam** [comment]
Hi @shanechambers-ds. Since I haven't heard anything from you in a while and because this PR was opened in mid-December and it's still open, I'm going to close it. You can re-open it when you have time to add the needed changes. Thank you!

**Pull request closed without merging**